fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractPLSearchTest.java

```diff
@@ -147,10 +148,7 @@ protected boolean searchReturnsResource(Class<? extends Resource> resourceTypeTo
         List<? extends Resource> resources = runQueryTest(resourceTypeToSearch, queryParms, Integer.MAX_VALUE);
         assertNotNull(resources);
         if (resources.size() > 0) {
```

Comment:
you could also remove this if and the final return false.  findResourceInResponse should handle empty list

Reply:
done